sdk/core/core-rest-pipeline/src/policies/proxyPolicy-react-native.mts

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/*
+ * NOTE: When moving this file, please update "browser" section in package.json
```

Comment:
should this comment be removed or updated to suit `tshy` config?

Reply:
This comment shouldn't be necessary, no.

Reply:
Fixed